risc0/bootstrap/poseidon/Cargo.toml

```diff
@@ -9,7 +9,7 @@ clap = { version = "4.0", features = ["derive"] }
 log = "0.4"
 regex = "1"
 risc0-core = { workspace = true }
-tempfile = "3.4"
+tempfile = "3.3"
```

Comment:
This was updated due to a security warning on this version.

Reply:
Hmmm that might be tricky to resolve due to a workspace conflict: https://github.com/risc0/risc0/pull/453/files#diff-a28f0c46e802066995e1c1431a4a3bb6bc88f01790bb9cb0200339bf4d22d826R13-R15

Reply:
Any suggestions on how to resolve?:

```
error: failed to select a version for `tempfile`.
    ... required by package `cargo-generate v0.18.0`
    ... which satisfies dependency `cargo-generate = "^0.18"` of package `cargo-risczero v0.13.0 (/home/user/src/risc0/risc0/cargo-risczero)`
versions that meet the requirements `~3.3` are: 3.3.0

all possible versions conflict with previously selected packages.

  previously selected package `tempfile v3.4.0`
    ... which satisfies dependency `tempfile = "^3.4"` of package `bootstrap-poseidon v0.1.0 (/home/user/src/risc0/risc0/bootstrap/poseidon)`

failed to select a version for `tempfile` which could resolve this conflict
```

Which is odd to me because I thought a workspace could hold multiple versions of a dep in it but I think it being a binary target causes some issue with the bootstrap target. Looks like `cargo-generate` has updated the tempfile dep to 3.4 in their tree but not published the change to crates.io yet.